packages/app-builder-lib/src/asar/asar.ts

```diff
@@ -66,13 +82,14 @@ export class AsarFilesystem {
     return result
   }
 
-  addFileNode(file: string, dirNode: Node, size: number, unpacked: boolean, stat: Stats): Node {
+  addFileNode(file: string, dirNode: Node, size: number, unpacked: boolean, stat: Stats, integrity?: NodeIntegrity): Node {
     if (size > 4294967295) {
       throw new Error(`${file}: file size cannot be larger than 4.2GB`)
     }
 
     const node = new Node()
     node.size = size
+    node.integrity = integrity
```

Comment:
Since `integrity?` is optional, should we first check it before setting it on `Node()`?
```
if (integrity) {
   node.integrity = integrity
}
```
Similar to how `node.unpacked` is beneath an if-statement (as opposed to being `node.unpacked = unpacked`)

Reply:
Ack.